frontend/src/Components/EventPreviewCard.jsx

```diff
@@ -118,7 +118,16 @@ export default function EventPreviewCard(props) {
                 }
                 <p><strong>Host: </strong>{event.instructor}</p>
                 <p><strong>For: </strong>{event.cohort}</p>
-                <p><strong>Location: </strong>{event.location}</p>
+
+                {
+                /* If location is a link for google meet or zoom, create a link for the location to open in a new tab
+                If not, leave the location as plan text */
+                
+                    event.location[0]==='z' || event.location[0]==='m'
```

Comment:
This can turn to a bug if the location is for example 'museum of moving images' or any other location starting by m or z
maybe we could check if the location starts with 'http' ? `event.location.slice(0, 4).toLowerCase() === 'http'`
what do you think?